### PR TITLE
Interests term display field

### PR DESCRIPTION
**Pull request by jmbarne3**
Enhancements:
* Added capitalization utility function to better support backup values for the `interests_display_text` meta field.
* Added some filter functions to `career_paths` to make it more consistent with what I'm doing on `interests`. Also updated comments on this class/functions.
* Added the `interests_display_text` meta field forms so they're baked into the plugin.

### CODE REVIEW

File: includes/ucf-degree-career-path-tax.php [hunk 1]
```diff
@@ -1,50 +1,87 @@
...
+		 * Retrieves the array of labels use in
+		 * registering the custom taxonomy
+		 * @author Jim Barnes
+		 * @since 0.0.1
+		 * @param array $labels The array of singular, plural and slug labels
+		 * @return array
+		 */
+		public static function labels( $labels ) {
+			$singular = $labels['singular'];
+            $plural   = $labels['plural'];
```
Comment (cjg89) [inline includes/ucf-degree-career-path-tax.php]: Mixed spaces/tabs on line 37

File: includes/ucf-degree-career-path-tax.php [hunk 2]
```diff
@@ -1,50 +1,87 @@
...
+				'labels'                     => self::labels( $labels ),
 				'hierarchical'               => false,
 				'public'                     => true,
 				'show_ui'                    => true,
 				'show_admin_column'          => true,
 				'show_in_nav_menus'          => true,
 				'show_tagcloud'              => true,
 			);
+
+			$retval = apply_filters( 'ucf_degree_career_path_taxonomy_args', $retval );
```
Comment (cjg89) [inline includes/ucf-degree-career-path-tax.php]: Recommend updating this filter name to `ucf_degree_career_paths_taxonomy_args` (pluralized "career_paths") for consistency with `ucf_degree_career_paths_labels`

File: includes/ucf-degree-interest-tax.php [hunk 3]
```diff
@@ -13,12 +13,108 @@ class UCF_Degree_Interest {
...
+		 * Forms fields to be added on the Add New Interest form
+		 * @author Jim Barnes
+		 * @since 3.1.0
+		 */
+		public static function add_meta_fields() {
+?>
+			<div class="form-field term-group">
+				<label for="interests_display_text"><?php _e( 'Display Text', 'ucf_degree' ); ?></label>
+				<input type="text" id="interests_display_text" name="interests_display_text">
+				<p class="help-text">The text that will be used when displaying Interests on the front end. If the interst you are adding has a comma, enter that here, not in the <code>Name</code> field above. If no text is provided, the value in the <code>Name</code> field will be used.</p>
```
Comment (cjg89) [inline includes/ucf-degree-interest-tax.php]: Typo "interst"

Comment (rjucf) [inline includes/ucf-degree-interest-tax.php]: interst misspelled

File: includes/ucf-degree-interest-tax.php [hunk 4]
```diff
@@ -47,8 +143,16 @@ public static function labels( $labels ) {
...
+		 * Returns the args array for registering
+		 * the custom taxonomy
+		 * @author Jim Barnes
+		 * @since 3.1.0
+		 * @param array $labels The array of singular, plural and slug labels
+		 * @return array
+		 */
         public static function args( $labels ) {
-            return array(
+            $retval = array(
```
Comment (cjg89) [inline includes/ucf-degree-interest-tax.php]: Tabbing is inconsistent

**Pull request merged**